echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/GithubNotificationAgent.java

```diff
@@ -101,7 +112,12 @@ public void sendNotifications(
 
     GithubStatus githubStatus = new GithubStatus(state, targetUrl, description, context);
     try {
-      githubService.updateCheck("token " + token, content.getRepo(), branchCommit, githubStatus);
+      final String repo = content.getRepo();
+      retrySupport.retry(
+        () -> githubService.updateCheck("token " + token, repo, branchCommit, githubStatus),
+        MAX_RETRY,
+        RETRY_BACKOFF,
+        true);
```

Comment:
We tend to not do exponential backoff in other areas of the code.

Reply:
Agree, I would recommend against using the exponential API.